WebWickerEiffel/src/main/java/com/tieto/webwicker/eiffel/web/commits/CommitPage.java

```diff
@@ -33,12 +33,12 @@
 import com.tieto.webwicker.eiffel.provider.CommitProvider;
 
 public class CommitPage extends WebWickerPage {
-	private static final long serialVersionUID = -1879929547602037385L;
-	private static final List<String> approved = Arrays.asList(new String[]{"+2"});
-	private static final List<String> partlyApproved = Arrays.asList(new String[]{"+1"});
-	private static final List<String> notApproved = Arrays.asList(new String[]{"-1","-2"});
-	private static final List<String> verified = Arrays.asList(new String[]{"+1"});
-	private static final List<String> notVerified = Arrays.asList(new String[]{"-1"});
+	
+	private static final List<String> approved = Arrays.asList(new String[]{"SUCCESS"});
+	private static final List<String> partlyApproved = Arrays.asList(new String[]{"INCONCLUSIVE"});
+	private static final List<String> notApproved = Arrays.asList(new String[]{"FAILURE"});
+	private static final List<String> verified = Arrays.asList(new String[]{"SUCCESS"});
+	private static final List<String> notVerified = Arrays.asList(new String[]{"INCONCLUSIVE"});
```

Comment:
Shouldn't it be INCONCLUSIVE and FAILURE? Or can FAILURE never occur for verified?